docs/SNAP_RECOMMENDATIONS.md

```diff
@@ -0,0 +1,146 @@
+# Recommendations
+
+While it is possible to construct recommendation controllers via the Snap configuration, it is recommended to utilize the [instantiator config](https://searchspring.github.io/snap/reference-snap-preact-instantiators#recommendationinstantiatorconfig) instead for integration of recommendations. This is because certain pages may not have a script block to target, and the instantiator will handle the targeting and creation of recommendation controllers for script blocks found in the DOM at run time.
```

Comment:
>This is because certain pages may not have a script block to target, and the instantiator will handle the targeting and creation of recommendation controllers for script blocks found in the DOM at run time.

^ can this be re-worded or clarified? Maybe we just need a brief description of what it does? Just a little hard for an uninitiated Snapophyte to follow I think.